### PR TITLE
lazyrules

### PR DESCRIPTION
**Pull request by celpjmnz**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
 // index.js
 
-
 alert("HELLO ITS MY FIRST COMMIT");
 
-//pull request
+function filter(res) {
+  if (isNaN(res)) {
+    return console.log("Just numbers");
```
Comment (VasylPosmit) [inline index.js:7]: use return OR console.log

File: index.js [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
+  } else {
+    if (res > 1000) {
+      return console.log("That's too much for me, sorry");
+    } else {
+      if (res % 1 === 0) {
+        let resF = res.toFixed();
```
Comment (VasylPosmit) [inline index.js:13]: Requirements are 
- if precise answer is fractions, than return "something around [number]"

**Review by VasylPosmit — CHANGES_REQUESTED** [review]

**Review by VladimirGOU — APPROVED** [review]

File: index.js [hunk 3]
```diff
@@ -1,7 +1,20 @@
 // index.js
 
-
 alert("HELLO ITS MY FIRST COMMIT");
 
-//pull request
```
Comment (VladimirGOU) [inline index.js:6]: can we delete this comment?

File: index.js [hunk 4]
```diff
@@ -1,7 +1,20 @@
...
+    if (res > 1000) {
+      console.log("That's too much for me, sorry");
+    } else {
+      if (res % 1 === 0) {
+        let resF = res.toFixed();
+        console.log("I don't know... Must be something around "+resF);
+      }
+    }
+  }
+}
```
Comment (TGacek89) [inline index.js:18]: Nice

**Review by TGacek89 — APPROVED** [review]

**Pull request closed without merging**